web/crux/prisma/migrations/20220815104826_url_prefix_to_image_name_prefix/migration.sql

```diff
@@ -0,0 +1,18 @@
+-- Copy data
+DROP TABLE IF EXISTS public._migration_instance;
+SELECT 
+	id,
+  "Registry"."urlPrefix"
+INTO public._migration_instance
```

Comment:
`_migration_registry` would be a better name

Reply:
true, my bad sry